spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :request do
```

Comment:
## Recommendation Change ♻️

- [ ] Let's modify the test name from UsersController to PostsController. It's a small change but I highly recommend you edit that 👍🏽

Reply:
Thank you for notifying me. I have done it. 
See: 
[Create rspec tests for posts controller2](https://github.com/MMGGYY66/rails-blog/pull/6/commits/91d9525f79e3b81e5f2bd2bea7f72119904b3891)